tests/baselines/reference/project/invalidRootFile/node/invalidRootFile.errors.txt

```diff
@@ -1,6 +1,8 @@
 error TS6053: File 'a.ts' not found.
 error TS6054: File 'a.t' has an unsupported extension. The only supported extensions are '.ts', '.tsx', '.d.ts'.
+error TS6229: No file under 'a' found with any supported extension.

```

Comment:
seems like the error shouldn't happen in the 'a.t' case, since we think '.t' is an extension

Reply:
@sandersn sorry, I'm not clear on the behavior change you're requesting. Only `'a'` is reported on right now - do you want others to have this error as well?

Alternately, in theory, some folks might be naming files like `contents.legacy.ts` and referring to them as `contents.legacy` - where `.legacy` could be misinterpreted as an extension... but the behavior now is to assume anything after a `.` is an extension.

Reply:
Well, looking back at it this week, I think I was confused and thought that both line 2 and 3 were new, but only line 3 is. So fixing an existing wrong error is out of scope for this PR -- sorry I suggested it.